Geocoder/src/main/java/com/example/Geocoder/configs/RateLimitConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.example.geocoder.configs;
+
+import java.time.Duration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.github.resilience4j.ratelimiter.RateLimiterConfig;
+import io.github.resilience4j.ratelimiter.RateLimiterRegistry;
+
+@Configuration
+public class RateLimitConfig {
+    
+    @Bean
+    public RateLimiterRegistry rateLimiterRegistry(){
+        RateLimiterConfig config = RateLimiterConfig.custom()
+            .limitForPeriod(1)
+            .limitRefreshPeriod(Duration.ofMinutes(10))
+            .timeoutDuration(Duration.ofMillis(10))
+            .build();
+        return RateLimiterRegistry.of(config);
+    }
```

Comment:
Issue: Typically we do not want to have multiple stories/tickets in a single PR.